rpc/src/main/scala/internal/service/service.scala

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.rpc.internal.service
+
+import java.io.{ByteArrayInputStream, InputStream}
+
+import freestyle.internal.ScalametaUtil
+import freestyle.rpc.protocol.{
+  BidirectionalStreaming,
+  RequestStreaming,
+  ResponseStreaming,
+  StreamingType
+}
+
+import scala.collection.immutable.Seq
+import scala.meta.Defn.{Class, Object, Trait}
+import scala.meta._
+import _root_.io.grpc.MethodDescriptor.Marshaller
+
+// $COVERAGE-OFF$ScalaJS + coverage = fails with NoClassDef exceptions
```

Comment:
Since rpc server implementation won't be compatible with ScalaJS, I think we could remove this annotation.